pkg/kando/location_pull.go

```diff
@@ -41,45 +36,20 @@ func newLocationPullCommand() *cobra.Command {
 		Args:  cobra.ExactArgs(1),
 		// TODO: Example invocations
 		RunE: func(c *cobra.Command, args []string) error {
-			return runLocationPull(c, args)
+			err := validateCommandArgs(c)
+			if err != nil {
+				return err
+			}
+			dataMover := NewDataMover(checkDataMover(c), "", kopiaSnapshotFlag(c))
+			destinationPath := pathFlag(c)
+			sourcePath := args[0]
+			return dataMover.Pull(sourcePath, destinationPath)
```

Comment:
```suggestion
			return dataMover.Pull(args[0], pathFlag(c))
```
do we really need to define it in new variable.